packages/astro/src/integrations/hooks.ts

```diff
@@ -327,6 +338,28 @@ export async function runHookConfigDone({
 						}
 						settings.adapter = adapter;
 					},
+					injectTypes(injectedType) {
+						if (integration.name === 'astro:db' && injectedType.filename === 'db-types.d.ts') {
+							const filename = './astro/db.d.ts';
```

Comment:
This feels like code smell, because we are handling something for `@astrojs/db`, an integration, inside core. Not sure why it's needed, but if we can't do otherwise, we should add a comment that explains this exception

Reply:
If we get rid of the code, the file will be generated at `.astro/integrations/astro_db/db-types.d.ts` which is not super pretty. Since `astro:db` is a core integration, there are already a few code smells here and there (eg. calling `getPackage`) so I think it's fine for now. I can add a comment to remember to remove it once `astro:db` is really part of core. Wdyt?

Reply:
Does the path need to be pretty? I’d think the path doesn’t really matter too much.